python/gemini-api/main.py

```diff
@@ -0,0 +1,14 @@
+import google.generativeai as genai
```

Comment:
Does this work?

```suggestion
import google.generativeai as genai
```

I am facing `ModuleNotFoundError: No module named 'google.generativeai'`